.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Tests
+
+on: [push]
+
+jobs:
+  test:
+    name: Node.js ${{ matrix.node_version }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node_version:
+          - 13
+          - 12
+          - 10
+        os:
+          - ubuntu-latest
+          - windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node_version }}
+      - run: npm install
+      - run: npm run test-coverage
+      - uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.github_token }}
+          parallel: true
+      - uses: coverallsapp/github-action@master
+        with:
+          github-token: ${{ secrets.github_token }}
+          parallel-finished: true
```

Comment:
One my experience, this not right, seem need  another job wait for matrix, not sure. I tried in this way before.

So I dont use  parallel https://github.com/fisker/id-placeholder/blob/ae506a0069bb459f6100f2c317f377ba9f8651de/.github/workflows/continuous-integration.yml#L52

Also, this action some times fail for no reason, issue can found on their repo, so I use continue-on-error

Reply:
The `parallel-finished` is what they document should be used when running multiple tests in parallel: https://github.com/coverallsapp/github-action#inputs

I added the `continue-on-error` on error now 👍 Good feedback!